### PR TITLE
Feat: Add test for maximum filesize issue #15

### PR DESCRIPTION
**Pull request by rbillings**
Added test verifying maximum filesize for local safebrowsing folders. Able to parse by name and specific filesize maximum. Addresses Issue #15 

Will refine test later with specific location of safebrowsing folder and add more tests for specific list file names.

### CODE REVIEW

**Comment by rpappalax** [comment]
@rbillings are you bringing in all these files temporarily just to get an mvp test stood up?

**Comment by rbillings** [comment]
Yes it was to be able to use the test in question - I thought maybe I'd use it for future tests. However now I am going to do a 180 and remove them!

File: tests/test_filesize.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+dir = ('safebrowsing')
+filenames = []
+for file in os.listdir(dir):
+        if os.path.isfile(os.path.join(dir, file)) and 'mozstd-trackwhite' in file:
+            size = os.path.getsize(os.path.join(dir, file))
+            if size <= 200000:
+                filenames.append(file)
+            else:
+                print file + ' is over maximum filesize'
```
Comment (rpappalax) [inline tests/test_filesize.py]: even if you're not using python3, best to make your code python3-compliant:
https://docs.python.org/3/whatsnew/3.0.html
use only print('bla bla') not print 'bla bla'

File: tests/test_filesize.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+print filenames
```
Comment (rpappalax) [inline tests/test_filesize.py]: this is good for first pass, but you'll definitely want to add asserts here to p/f your test

File: tests/test_filesize.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import os
+from os.path import getsize
+
+
+dir = ('safebrowsing')
+filenames = []
+for file in os.listdir(dir):
+        if os.path.isfile(os.path.join(dir, file)) and 'mozstd-trackwhite' in file:
+            size = os.path.getsize(os.path.join(dir, file))
+            if size <= 200000:
```
Comment (rpappalax) [inline tests/test_filesize.py]: perhaps it's just a style preference for tests, but you may want to create a constant for max size.

File: tests/test_filesize.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import os
+from os.path import getsize
+
+
+dir = ('safebrowsing')
+filenames = []
+for file in os.listdir(dir):
+        if os.path.isfile(os.path.join(dir, file)) and 'mozstd-trackwhite' in file:
```
Comment (rpappalax) [inline tests/test_filesize.py]: it seems like there will be at least one of these kinds of tests for each of the shavar files.  does the list of files already exist in a config somewhere that you might iterate over them one by one and check their filesize? (methinks that info is only in the testplan).
If I'm not mistaken, the challenge there will be that Francois had said certain file types should be <= a certain size and others would be different.  It may not hurt to add the filesizes to the prefs.ini, then ALL the info you need for your tests could be gotten from there.
Just my 2cents

Comment (rbillings) [inline tests/test_filesize.py]: There are different file size requirements for different file types. I was expecting to create multiple tests for different groupings of file size expectations. I'm not sure how adding the file size to the prefs.ini would help- but we can discuss face to face.

File: tests/test_filesize.py (hunk 3 above)
Comment (rbillings) [inline tests/test_filesize.py]: I'm not sure how I'd do that as the maximum size will be different depending on the list/file type.

File: tests/test_filesize.py (hunk 1 above)
Comment (rbillings) [inline tests/test_filesize.py]: Thanks for the tip! I'm not used to thinking about Python3 so I'll  make sure to change this.

**Comment by rpappalax** [comment]
This is 'rough' but we wanna get it into master, so MAKE IT SO
